### PR TITLE
Remove `GetAppSessions` and deprecated `DatabaseServerSpecV3` fiels

### PR DESCRIPTION
**Pull request by tigrato**
This PR removes `GetAppSessions` functions in favor of `ListAppSessions`.

It also removed all the deprecated fields in the `DatabaseServerSpecV3` message.

### CODE REVIEW

File: lib/web/apiserver_test.go [hunk 1]
```diff
@@ -4476,19 +4487,39 @@ func TestApplicationWebSessionsDeletedAfterLogout(t *testing.T) {
 		require.NoError(t, err)
 	}
 
+	collectAppSessions := func(ctx context.Context) []types.WebSession {
+		var (
+			nextToken string
+			sessions  []types.WebSession
+		)
+
+		// Leverages ListAppSessions instead of GetAppSessions to prevent
```
Comment (zmb3) [inline lib/web/apiserver_test.go:4496]: `GetAppSessions` no longer exists, so we can probably remove the comment

File: api/types/databaseserver.go [hunk 2]
```diff
@@ -287,14 +243,14 @@ func (s *DatabaseServerV3) GetAllLabels() map[string]string {
...
+	var dynLabels map[string]CommandLabelV2
 	if s.Spec.Database != nil {
 		for name, value := range s.Spec.Database.Metadata.Labels {
 			staticLabels[name] = value
 		}
+		dynLabels = s.Spec.Database.Spec.DynamicLabels
 	}
-
-	return CombineLabels(staticLabels, s.Spec.DynamicLabels)
+	return CombineLabels(staticLabels, dynLabels)
```
Comment (GavinFrazar) [inline api/types/databaseserver.go:253]: nit: can we simplify like this:

```go
if s.Spec.Database != nil {
  return CombineLabels(s.Metadata.Labels, s.Spec.Database.GetAllLabels())
}
return s.Metadata.Labels
```

**Review by GavinFrazar — APPROVED** [review]

**Review by zmb3 — APPROVED** [review]

**Review by smallinsky — APPROVED** [review]
LGTM Thanks for the cleanup.

**Comment by tigrato** [comment]
@zmb3 can you please skip the flaky tests detector?

**Pull request merged**